mimeparse.py

```diff
@@ -74,8 +74,11 @@ def parse_media_range(range):
     necessary.
     """
     (type, subtype, params) = parse_mime_type(range)
-    if 'q' not in params or not params['q'] or \
-            float(params['q']) > 1 or float(params['q']) < 0:
+    params.setdefault('q', params.pop('Q', None))  # q is case insensitive
+    try:
+        if not params.get('q') or not 0 <= float(params['q']) <= 1:
```

Comment:
Pretty minor, but we can use `params['q']` here instead of `params.get('q')` because `params.setdefault(...)` above ensures that `'q'` exists in the `params` dict.